cluster/addons/cluster-loadbalancing/glbc/glbc-controller.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: l7-lb-controller
+  name: l7-lb-controller-v0.5.2
```

Comment:
in the name? The image tag is 0.5.2, this is the pattern followed by other addons as well https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-rc.yaml.in#L4. Or did i misunderstand? 

We've alreayd cherry picked the tag bump in: https://github.com/kubernetes/kubernetes/pull/20535